LibreNMS/OS/Ifotec.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * ifotec.inc.php
+ *
+ * LibreNMS os poller module for Ifotec devices
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  LibreNMS contributors
+ * @author     Cedric MARMONIER
+ */
+
+namespace LibreNMS\OS;
+
+use Illuminate\Support\Str;
+use LibreNMS\Interfaces\Discovery\OSDiscovery;
+use LibreNMS\OS;
+
+class Ifotec extends OS implements OSDiscovery
+{
+    public function discoverOS(): void
+    {
+        $device = $this->getDeviceModel();
+
+        if (Str::startsWith($device->sysObjectID, '.1.3.6.1.4.1.21362.100.')) {
+            $ifoSysProductIndex = snmp_get($this->getDevice(), '.1.3.6.1.4.1.21362.101.1.1.0', '-Oqv');
+
+            if ($ifoSysProductIndex !== null) {
+                $oids = [
+                    '21362.101.1.2.1.5.' . $ifoSysProductIndex,
+                    '21362.101.1.2.1.7.' . $ifoSysProductIndex,
+                    '21362.101.1.2.1.8.' . $ifoSysProductIndex
+                ];
+                $oids2 = [
+                    '.1.3.6.1.4.1.' . $oids[0],
+                    '.1.3.6.1.4.1.' . $oids[1],
+                    '.1.3.6.1.4.1.' . $oids[2]
+                ];
+                $data = snmp_get_multi($this->getDevice(), $oids2, ['-OQUs']);
+                $device->version  = $data[$oids[1]]['enterprises'] . " (Bootloader " . $data[$oids[2]]['enterprises'] . ")";
```

Comment:
You're parsing the wrong values, this is what im seeing:
```
array:3 [
  "3.6.1.4.1.21362.101.1.2.1.5.1" => array:1 [
    "iso" => "0051"
  ]
  "3.6.1.4.1.21362.101.1.2.1.7.1" => array:1 [
    "iso" => "1.0.5d"
  ]
  "3.6.1.4.1.21362.101.1.2.1.8.1" => array:1 [
    "iso" => "x.x.x"
  ]
]
```

Reply:
I don't have the smae thing : 
`array(3) {
  ["21362.101.1.2.1.5.1"]=>
  array(1) {
    ["enterprises"]=>
    string(4) "0051"
  }
  ["21362.101.1.2.1.7.1"]=>
  array(1) {
    ["enterprises"]=>
    string(6) "1.0.5d"
  }
  ["21362.101.1.2.1.8.1"]=>
  array(1) {
    ["enterprises"]=>
    string(5) "x.x.x"
  }
}`